simpleflow/activity.py

```diff
@@ -87,7 +87,25 @@ def name(self):
         return '.'.join([prefix, name])
 
     def __repr__(self):
-        return 'Activity(name={}, version={}, task_list={})'.format(
+        return 'ActivityType(name={}, version={}, task_list={})'.format(
             self.name,
             self.version,
             self.task_list)
+
+
+class Activity(object):
```

Comment:
Yes, but currently in SWF executor, `ActivityTask` is created inside the submission to append related workflow/task credentials. 
Groups (and so their inside tasks) should not have be aware of the executor before being submitted.